diode-server/ingester/component.go

```diff
@@ -196,3 +203,13 @@ func validateRequest(in *diodepb.IngestRequest) error {
 
 	return nil
 }
+
+func newAuthUnaryInterceptor(authorizer authutil.Authorizer) grpc.UnaryServerInterceptor {
+	return func(ctx context.Context, req interface{}, _ *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
+		if err := authorizer.RequireScopesContext(ctx, []string{"diode:ingest"}); err != nil {
```

Comment:
Would be great to extract all supported scopes possibly to authutil as constants and use in all these changes replacing hardcoded strings